packages/www/layouts/streamDetail.tsx

```diff
@@ -71,7 +71,7 @@ const StreamDetail = ({
           ingest.sort((a, b) => a.base.localeCompare(b.base));
         }
         if (ingest.length > 0) {
```

Comment:
Should this have ? as well? If ingest is null I guess it will explode here first